src/openrct2/scenario/ScenarioRepository.h

```diff
@@ -42,7 +42,7 @@ enum class ScenarioSource : uint8_t
 
 struct ScenarioIndexEntry
 {
-    utf8 Path[MAX_PATH];
+    u8string Path;
```

Comment:
does this get serialised the same. I'm never 100% sure.

Reply:
It uses the DataSerializer and that handles char[size] and u8string the same, haven't noticed any differences.